tests/unit/test_pathutil.py

```diff
@@ -233,6 +236,7 @@ def test_make_localhost_symlinks_calls_make_symlink_for_each_key_value_dir(
         'share': '$DEE/suite3/share'}
     mocked_get_workflow_run_dir.return_value = "rund"
     mocked_expandvars.return_value = "expanded"
+    mocked_make_symlink.return_value = True
```

Comment:
Is this needed?

Reply:
The `make_symlink` function now returns true/false. Not sure if there is better way of doing it but I think I'd expect to be mocking this return?
The test is checking that `make_symlink` is called for all given directories, the return True has no bearing on the test (False would be grand too), other than to stop it blowing up.

Reply:
Strange, I would swear the line wouldn't be needed, because `make_localhost_symlinks()` still returns regardless. But the test does indeed fail if I remove that line. However, if I replace the patch decoration with
```python
mocked_make_symlink = Mock()
monkeypatch.setattr('cylc.flow.pathutil.make_symlink', mocked_make_symlink)
```
then it doesn't need a return value attr and the test passes